_data/simple-icons.json

```diff
@@ -14998,7 +14998,7 @@
         },
         {
             "title": "Spotify",
-            "hex": "1DB954",
+            "hex": "1ed760",
```

Comment:
You've done this change in #12252. Could you revert it here?

Reply:
sure i'll do